app/web/components/LanguagePickerSettings.tsx

```diff
@@ -0,0 +1,149 @@
+const COMMUNITY_BUILDER_FORM_LINK =
+  "https://couchers.org/community-builder-form";
+
+interface ChangeLanguageFormData {
+  newLanguage: string;
+}
+
+interface ChangeLanguageProps {
+  // language: string;
+  className?: string;
+}
+
+// TODO: this function is used in two places, where should it live?
+// note: this will not retrieve secure cookies in development,
+// manually add a non-secure cookie in development to test this functionality
+function getLangCookie() {
+  const name = "couchers-preferred-language" + "=";
+  const decodedCookie = decodeURIComponent(document.cookie);
+  const ca = decodedCookie.split(";");
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
+    while (c.charAt(0) == " ") {
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
+  }
+  return "";
+}
+
+export default function LanguagePickerSettings({
+  className,
+}: ChangeLanguageProps) {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+  const formClasses = useChangeDetailsFormStyles();
+  const theme = useTheme();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
+
+  console.dir(service);
+
+  const {
+    handleSubmit,
+    register,
+    reset: resetForm,
+  } = useForm<ChangeLanguageFormData>();
+  const onSubmit = handleSubmit(({ newLanguage }) => {
+    // TODO: send request to update cookie on backend w/ newLanguage code
+  });
+
+  const {
+    error: changeLanguageError,
+    isLoading: isChangeLanguageLoading,
+    isSuccess: isChangeLanguageSuccess,
+    mutate: changeLanguage,
+  } = useMutation<Empty, RpcError, ChangeLanguageFormData>(
+    ({ newLanguage }) => service.account.changeLanguage(newLanguage),
+    {
+      onSuccess: () => {
+        resetForm();
+      },
+    },
+  );
+
+  return (
+    <div className={className}>
+      {/* <Typography variant="h2">{t("auth:change_language_form.title")}</Typography> */}
+      <Typography variant="h2">Language</Typography>
+      <>
+        <Typography variant="body1">
+          {/* <Trans
+            i18nKey="auth:change_language_form.current_language_message"
+            values={{ language }}
+          >
+            {`Your language preference is currently `}
+            <strong>{language}</strong>
+            {`.`}
+          </Trans> */}
+          {`Your language preference is currently `}
+          <strong>{LANGUAGE_MAP[getLangCookie()].name}</strong>
+        </Typography>
+        <Typography
+          variant="body1"
+          paragraph
+          // className={classes.createCommunityText}
+        >
+          {/* <Trans i18nKey="dashboard:your_communities_helper_text2"> */}
+          {/* {`Don't see your community? `} */}
+          <MuiLink
+            href={COMMUNITY_BUILDER_FORM_LINK}
+            target="_blank"
+            rel="noreferrer noopener"
+            underline="hover"
+          >
+            <strong>Help translate couchers into your language.</strong>
```

Comment:
Same here, any hardcoded English should be switched into i18n json (this allows them to be switched into other languages).